mac 10.3 make failed

# cd github.com/TrueFurby/go-callvis/ && make
go get github.com/golang/dep/cmd/dep
dep ensure -update
make: dep: No such file or directory
make: *** [dep] Error 1

cd go-callvis/ && make
-------------------------
go get -u github.com/golang/dep/cmd/dep
dep ensure -update
make: dep: No such file or directory
make: *** [dep] Error 1

Still issues on macOS 10.12
```
$ go get -u github.com/TrueFurby/go-callvis
$ cd $GOPATH/src/github.com/TrueFurby/go-callvis && make
go get -u github.com/golang/dep/cmd/dep
dep ensure -update
make: dep: No such file or directory
make: *** [dep] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@ LDFLAGS := -X main.Version=$(VERSION)
 default: install
 
 dep:
-	dep ensure
+	$(GOPATH)/bin/dep ensure
 
 get-deps:
 	go get -u github.com/golang/dep/cmd/dep

